src/js/_enqueues/admin/common.js

```diff
@@ -2220,8 +2220,14 @@ $( function( $ ) {
 	// Listen for jQuery AJAX events.
 	( function( $ ) {
 		$( document ).ajaxComplete( function( event, xhr, settings ) {
+
+			// Return early if this is not the plugin-install.php page.
+			if ( window.pagenow !== 'plugin-install' ) {
+				return;
+			}
+
```

Comment:
I'm not convinced this change fixes the problem. The ajaxComplete is still listened to; it just exits under different conditions. Wouldn't it make more sense to move the page check as a wrapper before the listener?

```
if ( window.pagenow === 'plugin-install' ) {
    $( document ).ajaxComplete( function( event, xhr, settings ) {
        // etc.
    });
}
```